state/stateConsensus.go

```diff
@@ -255,11 +255,8 @@ func (s *State) Process() (progress bool) {
 	// is so we don't conflict with past version of the network if we have to reboot the network.
 	var Leader bool
 	var LeaderVMIndex int
-	if s.CurrentMinute > 9 {
```

Comment:
I first thought this was needed as a minute ceiling, but the function `GetVirtualServers` does this, so doing it outside is redundant :+1: